server/datastore/mysql/software.go

```diff
@@ -306,6 +307,19 @@ func selectSoftwareSQL(hostID *uint, opts fleet.SoftwareListOptions) (string, []
 		)
 	}
 
+	if opts.WithHostCounts {
```

Comment:
Are we sure that the UI will call with `opts.WithHostCounts=true` only after the vulnerability cron has executed at least once? (Otherwise no software will be returned, as there are no counts).

Reply:
This is not an option that is exposed to the UI, it's automatically set to true only in the right service call.

Reply:
OK yes just saw it, `ListSoftware` then will only return software after vulnerability processing has run at least once, right?

Reply:
Yep.